### PR TITLE
Add Twitter

### PR DESCRIPTION
**Pull request by Ana06**
openSUSE is really modern and it is using Twitter! Let new comers know about it :bowtie: 

image

image

image

### CODE REVIEW

File: _includes/gsoc-howto.html [hunk 1]
```diff
@@ -12,11 +12,14 @@ <h2 class="section-heading">How does it work?</h2>
...
           <p>
             <i class="fa fa-search fa-fw icon-mentoring"></i>
-            Select one of our mentoring <a href="#projects">projects</a>.
+            Select one of our mentoring <a href="#projects">projects</a> and follow the get started instructions in the project description.
           </p>
           <p>
             <i class="fa fa-comments fa-fw icon-mentoring"></i>
-            Get in touch with the mentor, follow the get started instructions in the project description.
+            Ask questions in our <a href="mailto: opensuse-project@opensuse.org">mailing list</a>,
+            <a href="" target="_blank">Twitter</a> and
```
Comment (foursixnine) [inline _includes/gsoc-howto.html]: Isn't it missing the twitter link?

Comment (ChrisBr) [inline _includes/gsoc-howto.html]: s/and/or/ ?

File: _includes/mentee-guidelines.html [hunk 2]
```diff
@@ -24,7 +24,8 @@ <h3>2. Help us to get to know you</h3>
   aspect of this process. Introduce yourself on our
   <a href="mailto: opensuse-project@opensuse.org">mailing list</a>
   ("Hey I'm thinking about becoming a mentee in 101 and I'm not
-  sure about XXX and have questions about YYY"), chat with us on
+  sure about XXX and have questions about YYY"), write as in
```
Comment (foursixnine) [inline _includes/mentee-guidelines.html]: I would use "get in touch with us on.. " or "Tweet us at @opensusementors", but in any case "write as in" sounds like instructions on how to write.

Comment (ChrisBr) [inline _includes/mentee-guidelines.html]: s/ write as/write us/

File: _includes/gsoc-howto.html (hunk 1 above)
Comment (Ana06) [reply _includes/gsoc-howto.html]: ouch! 🙈

Comment (ChrisBr) [reply _includes/gsoc-howto.html]: I would also not write twitter but the twitter account directly (@...)

File: _includes/mentee-guidelines.html (hunk 2 above)
Comment (Ana06) [reply _includes/mentee-guidelines.html]: _tweet us at @opensusementors_ sounds great!

**Comment by Ana06** [comment]
changes made! 😉

**Pull request merged**